Assets/Materials.meta

```diff
@@ -0,0 +1,9 @@
+fileFormatVersion: 2
```

Comment:
This meta file kept appearing for me, so I'm guessing it wasn't added previously?